unseal.go

```diff
@@ -68,16 +68,17 @@ import (
 // condition can also occur as the result of an incorrectly provisioned TPM, which will be detected during a subsequent call to
 // SealKeyToTPM.
 //
-// On success, the unsealed cleartext key is returned.
-func (k *SealedKeyObject) UnsealFromTPM(tpm *TPMConnection, pin string) ([]byte, error) {
+// On success, the unsealed cleartext key is returned as the first return value, and the private part of the key used for
+// authorizing PCR policy updates with UpdateKeyPCRProtectionPolicy is returned as the second return value.
+func (k *SealedKeyObject) UnsealFromTPM(tpm *TPMConnection, pin string) ([]byte, []byte, error) {
```

Comment:
now probably this should have named return arguments for clarity

Reply:
I've added named return arguments now, but I've also added a TPMPolicyAuthKey type which probably makes this unnecessary.